.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,34 @@
+---
+name: bug
+about: Create a report to help us improve
+title: 'bug: '
+labels: bug
+assignees: ''
+
+---
+
+**Description**
+A clear and concise description of what the bug is.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Screenshots/Code**
+If applicable, add screenshots or code to help explain your problem.
+
+**Additional context**
+Add any other context about the problem here.
+
+**Checklist**
+- [ ] git pull
+- [ ] make clean
+- [ ] make dirclean
+- [ ] make distclean
+- [ ] no related issue
```

Comment:
```suggestion
```

Reply:
either the bug is reproducable or not.
A check on related issues has no meaning at all